aeon_internal.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+import "aeon_error.proto";
+import "aeon_value.proto";
+
+package aeon;
+
+// Internal API to Aeon - a distributed database based on Tarantool.
+service InternalService {
+  // Get information about all instances.
+  rpc GetInfo(GetInfoRequest) returns (GetInfoResponse) {}
+}
+
+// Get information about all instances.
+
+message GetInfoRequest {}
+
+message GetInfoResponse {
+  // Error information. Set only on failure.
+  Error error = 1;
+  // Information of instances.
+  Value info = 2;
```

Comment:
may be it is worth to use sring (yaml/json) instead Value?

json is easy unmarshaled to structure, but `any` isn't.

Reply:
For now it was decided not to convert to json/yaml.